ext/phar/stream.c

```diff
@@ -44,7 +44,11 @@ php_stream_wrapper_ops phar_stream_wops = {
 	phar_wrapper_unlink,   /* unlink */
 	phar_wrapper_rename,   /* rename */
 	phar_wrapper_mkdir,    /* create directory */
-	phar_wrapper_rmdir,    /* remove directory */
+	phar_wrapper_rmdir     /* remove directory */
+#ifdef PHP_STREAMS_SUPPORT_IS_CACHEABLE
```

Comment:
I don't see why you need this ifdef if it's always going to be available... now you also have to mess up the codeing standards (, goes at the end).